game/scripts/npc/items/item_sange_and_yasha.txt

```diff
@@ -78,7 +78,7 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "movement_speed_percent_bonus"                    "8 12 16 20 24" //OAA
+        "movement_speed_percent_bonus"                    "20 25 30 35 40"
```

Comment:
In dota level 1 yasha combos have 30 MS level 1. You may have to put //OAA infront of it or make the scaling "30 35 40 45 50" or something around that.